packages/actions/src/releasePackages/action.yml

```diff
@@ -11,6 +11,9 @@ inputs:
     description: 'The published name of a single package to release'
   exclude:
     description: 'Comma separated list of packages to exclude from release (if not depended upon)'
+  main:
+    description: 'Is this running on the main branch?'
```

Comment:
Is it me, or this description doesn't match the usage?

> `if: ${{ !inputs.ref || inputs.ref == 'main' }}`
> ...
> `main: false`

Reply:
flipped the logic, thanks